README.md

```diff
@@ -60,180 +28,129 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+# 📖 [Personal_Portfolio] `<a name="about-project"></a>`
 
-## 🛠 Built With <a name="built-with"></a>
+**[Personal_Portfolio]** is a personal project to show my skills as a Software Developer
 
-### Tech Stack <a name="tech-stack"></a>
+## 🛠 Built With `<a name="built-with"></a>`
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+### Tech Stack `<a name="tech-stack"></a>`
```

Comment:
- [ ] Your don't need all the `<a>` tags infront of each headings on your README file. Kindly remove them.

![your-readme](https://github.com/ommarz/Personal_Portfolio/assets/26597432/10c0c95e-70ea-4e78-bae0-cbb0774442b2)